viadot/task_utils.py

```diff
@@ -342,6 +346,111 @@ def df_to_dataset(
     )
 
 
+class EnsureDFColumnOrder(Task):
+    """
+    Task for checking the order of columns in the loaded DF and in the SQL table into which the data from DF will be loaded.
+    If order is different then DF columns are reordered according to the columns of the SQL table.
+    """
+
+    def __init__(
+        self,
+        table: str = None,
+        schema: str = None,
+        if_exists: Literal["fail", "replace", "append", "delete"] = "replace",
+        df: pd.DataFrame = None,
+        dtypes: Dict[str, Any] = None,
+        config_key: str = None,
+        driver: str = "ODBC Driver 17 for SQL Server",
+        *args,
+        **kwargs,
+    ):
```

Comment:
Add docstring

Reply:
Is a docstring here necessary? There is a short docstring in linie 350 and another one in run function

Reply:
I think you should document your arguments here (or above). The function that has been run will be visible after calling `.run ()`. This docstring will be invisible to the user when the object is created.